feat(i18n): 调整语言切换功能并更新相关文案

dk333 opened the pull request:
- 在 Settings 组件中添加语言切换事件处理
- 更新英文、日文、中文语言文件中的"Language"相关文案
- 移除各语言文案中关于重启的提示信息

dk333 committed:
feat(i18n): 实现语言切换功能并更新相关文案

- 在 Settings 组件中添加语言切换事件处理
- 更新英文、日文、中文语言文件中的"Language"相关文案
- 移除各语言文案中关于重启的提示信息

Bronya0 reviewed (commented):
![局部截取_20250419_185619](https://github.com/user-attachments/assets/406cc508-1cf8-4168-a6f0-5a8750db879a)
这里没实现切换

dk333 committed:
feat(i18n): 添加语言切换时更新 Header 组件中的版本描述

- 在 Header 组件中添加了对语言切换的监听
- 当语言变化时，重新设置 subtitle 以反映最新的翻译
- 这个改动确保了在切换语言后，Header 组件中的版本描述能够正确更新
